_posts/2021-11-8-datafusion-6.0.0.md

```diff
@@ -0,0 +1,129 @@
+performance, easy to connect, easy to embed, and high quality.
+
+The Apache Arrow team is pleased to announce the DataFusion 6.0.0 release. This covers 4 months of development work
+and includes 122 commits from the following 28 distinct contributors.
+
+```
+# TODO update when we have a final 6.0 tag
+git shortlog -sn 5.0.0..87c8eaa datafusion datafusion-cli datafusion-examples
+    28  Andrew Lamb
+    25  Jiayu Liu
+     9  rdettai
+     8  QP Hou
+     5  carlos
+     4  Daniël Heres
+     4  Guillaume Balaine
+     4  Matthew Turner
+     4  Carlos
+     3  Francis Du
+     3  Jon Mease
+     3  Nga Tran
+     3  Marco Neumann
+     2  Andy Grove
+     2  Ruihang Xia
+     2  Yijie Shen
+     2  baishen
+     1  Krisztián Szűcs
+     1  Antoine Wendlinger
+     1  Qingping Hou
+     1  Conner Murphy
+     1  Taehoon Moon
+     1  Tiphaine Ruy
+     1  Jason Tianyi Wang
+     1  adsharma
+     1  Mike Seddon
+     1  Nan Zhu
+     1  Patrick More
+```
+
+<!--
+git log --pretty=oneline 5.0.0..87c8eaa datafusion datafusion-cli datafusion-examples | wc -l
+     122
+-->
+
+The release notes below are not exhaustive and only expose selected highlights of the release. Many other bug fixes
+and improvements have been made: we refer you to the complete
+[changelog](https://github.com/apache/arrow-datafusion/blob/6.0.0/datafusion/CHANGELOG.md).
+
+# New Website
+
+Befitting a growing project, DataFusion now has its
+[own website](https://arrow.apache.org/datafusion/) hosted as part of the
+main [Apache Arrow Website](https://arrow.apache.org)
+
+# Roadmap
+The community worked to gather their thoughts about where we are
+taking DataFusion into a public
+[Roadmap](https://arrow.apache.org/datafusion/specification/roadmap.html)
+for the first time
+
+# Performance
+TODO: Anything to report??
+
+# New Features
+
+- Support for `EXPLAIN ANALYZE` and a new runtime metric collection
+- DataFrame support for: `show`, `limit`,
+- Support for `trim ( [ LEADING | TRAILING | BOTH ] [ FROM ] string text [, characters text ] )` syntax
+- Support for Postgres style regular expression matching operators `~`, `~*`, `!~`, and `!~*`
+- Automatic schema inference for CSV files
+- Support for SQL set operators `UNION`, `INTERSECT`, and `EXCEPT`
+- `cume_dist`, `percent_rank`, Window Functions
+- `digest`, `blake2s`, `blake2b` functions
+- HyperLogLog based `approx_distinct`
+- `is distinct from` and `is not distinct from`
+- HIVE style partitioning support, for Parquet, CSV, Avro and Json files on local or remote storage
+- Generic constant evaluation and simplification framework
+- Support for `CREATE TABLE AS SELECT`
+- Better interactive editing support in `datafusion-cli` as well as `psql` style commands such as `\d`, `\?`, and `\q`
+- Support for accessing elements of `Struct` and `List` columns (e.g. `SELECT struct_column['field_name'] FROM ...`)
+
+
+# `async` Planning and Split File Format and Layout
```

Comment:
I think the related PR link will be generated in the 6.0.0 Changelog. But I feel this is a big change and great improvement so it might be helpful to have a direct pointer(apache/arrow-datafusion#1010 and maybe the design doc as well) in this blog.

Reply:
Great idea @capkurmagati  -- done in 9846e52c5

Reply:
cc @rdettai